css/project-list-pages.styl

```diff
@@ -43,6 +43,10 @@
     .pill-button
       margin-right: 10px
       padding: 1em
+      color: rgb(64,64,64)
+      font-weight: normal
+      border-color: rgb(64,64,64) !important
+      box-shadow: rgb(64, 64, 64) 0px 0px 1px 0px inset !important
```

Comment:
Could that be fixed by removing the extra `nav` element that's wrapped around the bottom pagination buttons? Might be a quicker fix than writing new CSS. Apart from that element, it looks like both navigation blocks have the same markup.